packages/grid/default.env

```diff
@@ -27,7 +27,8 @@ TRAEFIK_VERSION=v2.10
 REDIS_VERSION=6.2
 RABBITMQ_VERSION=3
 SEAWEEDFS_VERSION="3.59"
-DOCKER_IMAGE_SEAWEEDFS=chrislusf/seaweedfs
+DOCKER_IMAGE_SEAWEEDFS=openmined/grid-seaweedfs
+DOCKER_IMAGE_SEAWEEDFS_ORIGINAL=chrislusf/seaweedfs
```

Comment:
this is always the same I suppose? If thats true, why do we have a variable for it

Reply:
yes, I understand that, but why is it a variable, instead of hardcoded

Reply:
ya good point. not very useful here.